src/marqo/tensor_search/utils.py

```diff
@@ -139,6 +150,89 @@ def check_device_is_available(device: str) -> bool:
         return False
 
 
+def convert_to_MediaType(type_str: str) -> MediaType:
+    #This can actually be a dictionary
+    if type_str == "text":
+        return MediaType.text
+    elif type_str == "image":
+        return MediaType.image
+    elif type_str == "video":
+        return MediaType.video
+    else:
+        raise errors.MarqoError(f"Unspported type for {type_str}")
+
+def convert_to_FileType(type_str = "default") -> FileType:
+    #This can actually be a dictionary :(
+    if type_str == "default":
+        return FileType.default
+    elif type_str == "url":
+        return FileType.url
+    elif type_str == "www.youtube.com":
+        return FileType.youtube
+    elif type_str == "www.tiktok.com":
+        return FileType.tiktok
+    elif type_str == "local_path":
+        return FileType.local
+    elif type_str == "straight_text":
+        return FileType.straight_text
+    elif type_str == "ndarray":
+        return FileType.ndarray
+    elif type_str == "PILImage":
+        return FileType.PILImage
+    elif type_str == "ListOfPILImage":
+        return FileType.ListOfPILImage
+    else:
+        raise errors.MarqoError(f"Unsupported filetype for {type_str}")
+
+
+
+def get_filename_from_cd(cd):
+    """
+    Get filename from content-disposition
+    """
+    if not cd:
+        return None
+    fname = re.findall('filename=(.+)', cd)
+    if len(fname) == 0:
+        return None
+    return fname[0]
+
+def content_routering(field_content, download_path, infer_if_media = True):
```

Comment:
This is function acts like an interface, can you please make the function signature more detailed? And please add a docstring.



For example: 
```
def content_routering(field_content: str, download_path: str, infer_if_media = True) -> Tuple[str, str, str]
"""
Args:
   field_content: <PARAMETER DESCRIPTION>
   download_path: <PARAMETER DESCRIPTION>
    infer_if_media: : <PARAMETER DESCRIPTION>

Returns 
    <DESCRIPTION OF RETURNED VALUES> 
```

Reply:
solved. thanks for the comments! this makes the code much better.